source/collaborate/schedule-messages.rst

```diff
@@ -0,0 +1,43 @@
+Schedule messages
+==================
+
+.. include:: ../_static/badges/ent-pro-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+There are times when you want to send a message, but you don't want it sent immediately. For example, it may be after working hours for the recipient. 
+From Mattermost v10.3, using Mattermost in a web browser or the desktop app, you can schedule messages to send in the future. 
+
+Compose a message, then select the right side of the **Send** icon |send-icon| to schedule when the message will be sent. You can choose from a fixed time or set a custom time.
```

Comment:
```suggestion
Compose a message, then select the right side of the **Send** button |send-icon| to schedule when the message will be sent. You can choose from a fixed time or set a custom time.
```

Should we consider something more like this image? 
![image](https://github.com/user-attachments/assets/68ddefba-a3c8-458f-ae3b-83c546f5ca16)